notifier/bot/botController.py

```diff
@@ -0,0 +1,35 @@
+from telegram.ext import ( Application, CommandHandler, filters, MessageHandler
+)
+import logging
+from .handlers.commandHandlers import (start, help, list, track, untrack);
+from .handlers.errorHandlers import nonExistent;
+
+import os;
+from dotenv import load_dotenv;
+
+
+load_dotenv();
```

Comment:
don't ever use `.env` instead export the variable in your terminal then do the following
```python
import os
os.environ.get("YOUR_ENV_VAR"]
```
this is important so we can get those vars from Heroku env var or GutHub secrets for testings

Reply:
should I set it inline every time I call the bot in the terminal? can't we resort to this way when moving to production at least?

Reply:
yes u have to for every new terminal, but there are two solutions:

1 - export the secretes in the windows ENV always, on unix systems we add it to the file `.profile`
2 - a better sol, I just used VS Code extension called `secrets` it populate any new terminal with the secretes, I like more